src-lib/darknet.h

```diff
@@ -894,9 +877,9 @@ typedef struct detection{
 	float *mask;
 	float objectness;
 	int sort_class;
-	float *uc; // Gaussian_YOLOv3 - tx,ty,tw,th uncertainty
-	int points; // bit-0 - center, bit-1 - top-left-corner, bit-2 - bottom-right-corner
-	float *embeddings;  // embeddings for tracking
+	float *uc; /// Gaussian_YOLOv3 - tx,ty,tw,th uncertainty
```

Comment:
If you want the doxygen comment apply to the current line instead of the next line, you must use `///<` instead of `///`.